src/throttle/amoc_throttle_controller.erl

```diff
@@ -98,6 +85,13 @@ stop(Name) ->
 telemetry_event(Name, Event) when Event =:= request; Event =:= execute ->
     raise_event(Name, Event).
 
+-spec raise_event_on_slave_node(name(), event()) -> ok.
+raise_event_on_slave_node(Name, Event) ->
```

Comment:
Let's add a comment here that the purpuse of this function is to ensure that there are no event duplicates if we are running in a single node (non distributed) mode.